src/components/Button.tsx

```diff
@@ -0,0 +1,16 @@
+import Link from 'next/link';
+
+type ButtonProps = {
+  href: string;
+  title: string;
+};
+
+const Button = ({ href, title }: ButtonProps) => {
+  return (
+    <Link href={href}>
+    <a href={href} className="inline-block bg-black text-white py-2 px-6 rounded hover:bg-gray-700 transition duration-300">{title}</a>
+    </Link>
```

Comment:
I'm getting the following error when running the application. Apparently, Next.js doesn't like nesting an `<a>` tag within their `<Link>` component.
```
Error: Invalid <Link> with <a> child. Please remove <a> or use <Link legacyBehavior>.
```

This 👇  fixed the issue on my end
```suggestion
    <Link href={href} className="inline-block bg-black text-white py-2 px-6 rounded hover:bg-gray-700 transition duration-300">{title}</Link>
```